tests/unit/test_tfrecord_format.py

```diff
@@ -318,7 +318,7 @@ class TfrecordImporterTest(TestCase):
     @mark_requirement(Requirements.DATUM_GENERAL_REQ)
     def test_can_detect(self):
         detected_formats = Environment().detect_dataset(DUMMY_DATASET_DIR)
-        self.assertEqual([TfDetectionApiImporter.NAME], detected_formats)
+        self.assertIn(TfDetectionApiImporter.NAME, detected_formats)
```

Comment:
Why should it be changed? There is no way to differentiate between `roboflow_tfrecord` and `tf_detection_api` data formats?

Reply:
It seems that we can distinguish them with the existence of `"image/source_id"` field.
https://github.com/openvinotoolkit/datumaro/blob/f3803256ef35073359affbb34a9f79854353e2a3/src/datumaro/plugins/data_formats/tf_detection_api/base.py#L71-L72

Reply:
Yes, but, in order to do this, we have to decode tfrecord file.

Reply:
Can we do that? I expect that inspecting the schema of tfrecord file is not taking much time.

Reply:
I added at 2f73dff00ce74983daf1a477b9fbaec11843ecfa. Thanks.